samples/sidebar/index.html

```diff
@@ -101,6 +101,7 @@
         resize: 'detect',
         // sendTyping: true,    // defaults to false. set to true to send 'typing' activities to bot (and other users) when user is typing
         user: user,
+        locale: 'es-es', // override locale to Spanish
```

Comment:
This feels jammed in here because this isn't a sample made to highlight locale features. It also feels random.. why only this sample and not all of them? I agree it is a good idea to communicate through the samples that this parameter exists. I think it would be more consistent to add it to all samples, but commented out:
```ts
// locale: 'es-es', // optional: override locale
```

Reply:
Agree, fixed. 😄